pkg/helm/stack.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/okteto/okteto/pkg/cmd/build"
+	k8Client "github.com/okteto/okteto/pkg/k8s/client"
+	"github.com/okteto/okteto/pkg/k8s/namespaces"
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/okteto"
+)
+
+const (
+	// HelmDriver default helm driver
+	HelmDriver = "secrets"
+)
+
+//Translate translates the original stack and build images
+func Translate(s *model.Stack, forceBuild bool) error {
```

Comment:
We should split `Translate` and `Build` into two different functions, so it's obvious what's happening.